build/ci.go

```diff
@@ -270,7 +270,7 @@ func doTest(cmdline []string) {
 	build.MustRun(goTool("vet", packages...))
 
 	// Run the actual tests.
-	gotest := goTool("test", buildFlags(env)...)
+	gotest := goTool("test --timeout=30m", buildFlags(env)...)
```

Comment:
Could you append "timeout", "30m" in `gotest.Args`?
`gotest.Args = append(gotest.Args, "--timeout=30m"`

Reply:
Thank you! It has no errors now :)